sdccc/src/main/java/com/draeger/medical/sdccc/tests/glue/direct/DirectSubscriptionHandlingTest.java

```diff
@@ -83,47 +85,49 @@ public class DirectSubscriptionHandlingTest extends InjectorTestBase {
     public static final Duration DURATION = Duration.ofHours(1);
     public static final long TIMEOUT_NANOS = 5000 * NANOS_IN_A_MILLISECOND;
     public static final int POLLING_WAIT_TIME_MILLIS = 1000;
+    public static final String SUBSCRIPTION_END_STATUS_DELIVERY_FAILURE =
+            "http://schemas.xmlsoap.org/ws/2004/08/eventing/DeliveryFailure";
 
     private static final Logger LOG = LogManager.getLogger(DirectSubscriptionHandlingTest.class);
 
     private static final String ROOM1 = "123";
     private static final String ROOM2 = "124";
+    private final InterceptorWithTheAbilityToFailReports interceptor = new InterceptorWithTheAbilityToFailReports();
```

Comment:
Do not initialize InterceptorWithTheAbilityToFailReports here!

That is because subscribeToReport is called multiple times and currently it is only save due to addAbilityToFail being true for only one of these calls.

It may be a local.

Reply:
Indeed. corrected.